src/pages/en/guides/troubleshooting.mdx

```diff
@@ -137,6 +137,10 @@ nodeLinker: "node-modules"
 
 In Astro, using a `<head>` tag works like any other HTML tag: it does not get moved to the top of the page or merged with the existing `<head>`. Because of this, you usually only want to include one `<head>` tag throughout a page. We recommend writing that single `<head>` and its contents in a [layout component](/en/core-concepts/layouts/).
 
+### An unwanted `<script>` or `<style>` is included
+
+Astro's build process works on the module graph: this means that once a component is used, its `<script>` and `<style>` tags are processed, optimized, and bundled, whether it is rendered or not. This does not apply if the `is:inline` directive is applied. 
```

Comment:
```suggestion
Astro's build process works on the module graph: once a component is evaluated, its `<script>` and `<style>` tags are processed, optimized, and bundled, whether it is rendered or not. (Note: This does not apply to scripts when the `is:inline` directive is applied.)
```
1. the colon takes the place of needing to say "this means that" so we can shorten this!
2. I think "used" isn't quite descriptive enough. And, I think "used" could be interpreted as another way of saying "rendered" (e.g. I didn't *use* the component, so why do I see it's stuff??) so I'd like to avoid any chance of confusion. Can we think of a better word? I threw in "evaluated" as a placeholder, but what do we think is the best, descriptive verb here? "read" ? ... ?

Reply:
1. agreed!
2. since the wording is not 100% obvious to you guys either, how about going for "evaluated" adding a simple `(i.e. appears after the code fence)`